.travis.yml

```diff
@@ -50,14 +50,14 @@ addons:
 
 jobs:
   include:
-    - env: LUA=5.2 LUANAME=lua5.2 DO_COVERAGE=coveralls
+    - env: LUA=5.2 LUANAME=lua5.2 DO_COVERAGE=coveralls MANUAL_SCREENS=1
       addons:
         apt:
           packages:
             - *BASE_PACKAGES
             - liblua5.2-dev
             - lua5.2
-    - env: LUA=5.3 LUANAME=lua5.3 DO_COVERAGE=codecov
+    - env: LUA=5.3 LUANAME=lua5.3 DO_COVERAGE=codecov MANUAL_SCREENS=1
```

Comment:
I think it might make sense to turn "env" into a list, but not really necessary here already.